handout/components/app_structure/access_child_components_from_template.md

```diff
@@ -0,0 +1,38 @@
+# Access Child Components From the Template
+In our templates, we may find ourselves needing to access values provided by the child components which we use to build our own component.
+
+The most straightforward examples of this may be seen dealing with forms or inputs:
+
+_app/my-example.component.html_
+```html
+<section >
+  <form #myForm="ngForm" (ngSubmit)="submitForm(myForm)">
+    <!-- ... -->
+  </form>
+</section>
+```
+
+
+_app/my-example.component.ts_
+```ts
+// ...
+import {NgForm} from '@angular/forms';
+// ...
+export class MyExampleComponent {
+  submitForm (form: NgForm) {
+    // ...
+  }
+}
+```
+[View Example](https://plnkr.co/edit/MFaLMCbLOIXfNhYWT9ni?p=preview)
+
+This isn't just a magic feature which only forms have.
+
+```html
+<red-ball #myBall></red-ball>
```

Comment:
Wasn't really sure where the value for `myBall` is coming from so, made a plunker for it: https://plnkr.co/edit/UTBleHSpTf5To8zZSbSd?p=preview

IMO it's probably worth adding that in or something similar